package.json

```diff
@@ -289,5 +289,8 @@
   },
   "engines": {
     "node": ">=12 <13"
+  },
+  "volta": {
```

Comment:
What does this do? Is it for those that using volta to manage node versions?

Reply:
@marefr Good catch, thanks. I actually didn't mean to commit that one. I use it locally, since I use volta instead of nvm, but wasn't meaning to commit it :man_facepalming:

Reply:
If it adds value to volta users it's probably a good thing, but probably in a separate PR then.